lib/plots/pane.js

```diff
@@ -35,7 +35,9 @@ export default class PlotPane extends PaneItem {
       </div>,
       <Button icon='circle-slash' alt='Forget Plot' onclick={() => this.show()} />
     ];
-    if (this.item && this.item.toolbar) buttons.push(toView(this.item.toolbar))
+
+    if (this.item && this.item.toolbar) buttons = buttons.concat(toView(this.item.toolbar))
```

Comment:
Since we're fully expecting an array of views here, would be better to just `map`.